### PR TITLE
Add TS types

### PR DESCRIPTION
**Pull request by kibertoad**
refs https://github.com/pinojs/pino/pull/913
refs https://github.com/pinojs/pino/issues/910

### CODE REVIEW

**Comment by kibertoad** [comment]
@mcollina Could you also check this one out?

File: .travis.yml [hunk 1]
```diff
@@ -16,6 +22,7 @@ node_js:
 script:
   - npm run lint-ci
   - npm run test-ci
+  - ${TSSCRIPT:-echo "Skip TS tests"}
```
Comment (kibertoad) [inline .travis.yml:25]: This is needed because older Node versions that we still support are not compatible.

**Review by lastthyme — APPROVED** [review]

**Review by lastthyme — APPROVED** [review]

**Comment by kibertoad** [comment]
@mcollina This is ready for review.

**Review by mcollina — APPROVED** [review]
lgtm

File: package.json [hunk 2]
```diff
@@ -3,11 +3,22 @@
   "version": "2.5.0",
   "description": "A collection of standard object serializers for Pino",
   "main": "index.js",
+  "type": "commonjs",
+  "types": "index.d.ts",
+  "files": [
```
[the remark this answers is not in the document]
Comment (kibertoad) [reply package.json]: Removed.

**Comment by kibertoad** [comment]
@jsumners Is it OK now?

**Comment by jsumners** [comment]
Yes.

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**